modules/util/config/ConceptConfig.py

```diff
@@ -108,6 +108,7 @@ class ConceptConfig(BaseConfig):
     path: str
     seed: int
     enabled: bool
+    validation_enabled: bool
```

Comment:
suggestion for a better name: validation_concept
you don't enable or disable validation with this setting, you just define if this concept is used for validation